qute.ls/com.redhat.qute.ls/src/test/java/com/redhat/qute/services/completions/QuteCompletionInExpressionWithLetSectionTest.java

```diff
@@ -42,6 +42,21 @@ public void objectPart() throws Exception {
 				c("age", "age", r(3, 14, 3, 14)));
 	}
 
+	@Test
+	public void objectPartWithSpaces() throws Exception {
+		String template = "{@org.acme.Item item}\r\n" + //
+				"{#let myParent =  item.name         isActive =false age= 10} \r\n" + //
+				"  <h1>{myParent.name}</h1>\r\n" + //
+				"  Is active: {|}\r\n" + // <-- completion here
+				"  Age: {age}\r\n" + //
+				"{/let}";

```

Comment:
multi-ligne quotes

Reply:
We need to update Java version if we want to do that.

Reply:
OK. We should bump all LS to require Java 17 minimum